src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { createRef } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { LAPTOP_BP_UP } from '../blockHelpers'
+import { UnderlineComponent, UNDERLINE_HEIGHT } from './Underline'
+
+const TabContainer = styled.button<{ selected?: boolean }>`
```

Comment:
Do we override the focus state globally? Don't remember, otherwise would be nice to add one here :)

Reply:
You mean adding a focus state for the tabs? @robinandeer